<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_manage_Dialog</class>
 <widget class="QDialog" name="book_manage_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理</string>
  </property>
  <widget class="QGroupBox" name="group_booklist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>图书列表</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="userview">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="group_add_single">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>681</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>单册图书添加</string>
   </property>
   <widget class="QLineEdit" name="single_title">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>306</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>书目：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="single_type">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>175</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>175</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>类别：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="single_press">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>出版社：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>105</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>年份：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="single_year">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>105</y>
      <width>96</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>单价：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="single_price">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>96</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>175</y>
      <width>96</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>数量：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="single_add">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>96</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>105</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>作者：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="single_author">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>105</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="single_isbn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>35</y>
      <width>306</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>35</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="single_num">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>172</y>
      <width>96</width>
      <height>36</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <zorder>label</zorder>
   <zorder>single_type</zorder>
   <zorder>label_2</zorder>
   <zorder>single_press</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>single_year</zorder>
   <zorder>label_5</zorder>
   <zorder>single_price</zorder>
   <zorder>label_7</zorder>
   <zorder>single_add</zorder>
   <zorder>label_8</zorder>
   <zorder>single_author</zorder>
   <zorder>single_isbn</zorder>
   <zorder>label_9</zorder>
   <zorder>single_title</zorder>
   <zorder>single_num</zorder>
  </widget>
  <widget class="QGroupBox" name="group_nav">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>操作选项</string>
   </property>
   <widget class="QPushButton" name="guide_booklist">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guide_add_single">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加新书</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userview</tabstop>
  <tabstop>guide_booklist</tabstop>
  <tabstop>guide_add_single</tabstop>
  <tabstop>single_isbn</tabstop>
  <tabstop>single_title</tabstop>
  <tabstop>single_author</tabstop>
  <tabstop>single_year</tabstop>
  <tabstop>single_press</tabstop>
  <tabstop>single_price</tabstop>
  <tabstop>single_type</tabstop>
  <tabstop>single_num</tabstop>
  <tabstop>single_add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
